### ISSUE TITLE
option to see tab opened date/hour and site icon in results

### ISSUE THREAD

**Issue by mailinglists35** [janraasch/tab-ahead#14]
![image](https://cloud.githubusercontent.com/assets/2054302/4929955/879f0e24-655b-11e4-98da-5a4deda32d5f.png)

**Comment by janraasch** [comment]
Closing this as there has not been any action here in eight months.

**Comment by mailinglists35** [comment]
obviously I have no great graphical skills; the image was just a proposal.
let me rephrase: can you add the site icon and date, in any way you consider to be not horrible?

**Comment by janraasch** [comment]
Hi @mailinglists35. No worries. As I said:

> If somebody can come up with a visually pleasing version of this, maybe even a working html/css mock-up, I'd be all for it.

but nobody (including myself :)) has done this, which is why I closed the ticket.

**Comment by mailinglists35** [comment]
i'd be happy to have an ugly version rather not have one at all
